### ISSUE TITLE
Problem with non UTF-8 characters

### ISSUE THREAD

**Issue by jjoseba** [Sommerregen/grav-plugin-external-links#4]
When the link's text contains some non UTF-8 characters, they get corrupted somehow.
Example:

`[AáÉ ‘quote’ ü ¿question?](http://otherdomain.com/ "Title")`
produces
`<a href="http://otherdomain.com/" title="Title" target="_blank" rel="nofollow" class="external-link no-image">AÃ¡Ã âquoteâ Ã¼ Â¿question?</a>`

I thought it could be something related with my PHP config, but with the plugin disabled it works normally.
Cheers!

**Comment by Sommerregen** [comment]
Hi @jjoseba,

I tested your example and indeed produced this bug. Ironically you just have to uncomment line 404 in https://github.com/Sommerregen/grav-plugin-external-links/blob/master/classes/ExternalLinks.php#L404 to fix it.

It may lead to other problems, since in the past I commented out this line due some problems I experienced with it, I don't know anymore... It would be good, if you test this fix and leave me a note when getting other issues.

Thanks!
